adback-solution-to-adblock/includes/class-ad-back-activator.php

```diff
@@ -165,7 +165,11 @@ public static function initializeBlog()
             $notices = get_option('adback_deferred_admin_notices', array());
             $notices[] = sprintf(__('Registration error', 'adback-solution-to-adblock'), get_admin_url($blogId, 'admin.php?page=ab-settings'));
             update_option('adback_deferred_admin_notices', $notices);
+
+            $errorMsg = isset($data['error']['message']) ? $data['error']['message'] : 'error';
```

Comment:
maybe the check on the error messages `error` is a little light. maybe you could do a check on the error code which can be different for your exception